src/util/webextensionRPC.test.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-env jest */
+
+import { remoteFunction } from './webextensionRPC'
+
+describe('remoteFunction', () => {
+    test('should return a remotely callable function', () => {
+        const remoteFunc = remoteFunction('remoteFunc', {tabId: 1})
+        expect(remoteFunc.name).toBe('remoteFunc_RPC')
+    })
+
+    test('should throw an error when remoteFunc is called', async () => {
+        const remoteFunc = remoteFunction('remoteFunc', {tabId: 1})
```

Comment:
`expect.assertions(1)` is missing. Please check all tests where you expect in a catch block that they have `expect.assertions(...)` in the beginning.